pkg/db/writer/open.go

```diff
@@ -32,7 +32,12 @@ func (o config) ConnectionString() (string, error) {
 func open(cfg config) (*gorm.DB, error) {
 	if cfg.Overwrite {
 		// the file may or may not exist, so we ignore the error explicitly
-		_ = os.Remove(cfg.DbPath)
+		if _, err := os.Stat(cfg.DbPath); !os.IsNotExist(err) {
```

Comment:
do we have a test for this?

Reply:
we do not, but given that this is regarding testing side effects on disk this isn't easy to test. The code would need to get refactored to use a filesystem abstraction and setup a fake FS to test what the action would be. Since this was boyscouting a missing error and we're in a time crunch I'd vote not taking the time to do the above refactors to add a test at this time.

Reply:
No need to block on this, thanks for explaining